e3db/client.py

```diff
@@ -1193,7 +1194,7 @@ def write_file(self, record_type, plaintext_filename, plain={}):
         # pending file write created
         response_json = response.json()
         upload_file.file_url = response_json["file_url"]
-        upload_file.record_id = uuid.UUID(response_json["id"])
+        upload_file.record_id = response_json["id"]
```

Comment:
Does this get turned into a UUID automatically, or is it something that is really not needed in this instance?

Reply:
Yeah, we were doing an unnecessary step, which then stopped working in 3.